src/clusterfuzz/_internal/build_management/build_manager.py

```diff
@@ -605,10 +663,18 @@ def __init__(self,
                revision,
                build_url,
                build_prefix='',
-               fuzz_target=None):
+               fuzz_target=None,
+               http_build_url=None):
+    """RegularBuild constructor. See Build constructor for other parameters.
+
+    Args:
+        http_build_url: the http build URL. It differs from build_url as
+        build_url points to the GCP bucket URL. Defaults to None.
```

Comment:
I would give an explicit example: `gs://foo/bar.zip` - otherwise sure, they both are GCP bucket URLs, in a sense?